app-tests/run.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+set -e
+
+export OPAL_AUTH_PUBLIC_KEY
+export OPAL_AUTH_PRIVATE_KEY
+export OPAL_AUTH_MASTER_TOKEN
+export OPAL_CLIENT_TOKEN
+export OPAL_DATA_SOURCE_TOKEN
+
+function generate_opal_keys {
+  echo "- Generating OPAL keys"
+
+  ssh-keygen -q -t rsa -b 4096 -m pem -f opal_crypto_key -N ""
+  OPAL_AUTH_PUBLIC_KEY="$(cat opal_crypto_key.pub)"
+  OPAL_AUTH_PRIVATE_KEY="$(tr '\n' '_' < opal_crypto_key)"
+  rm opal_crypto_key.pub opal_crypto_key
+
+  OPAL_AUTH_MASTER_TOKEN="$(openssl rand -hex 16)"
+  OPAL_AUTH_JWT_AUDIENCE=https://api.opal.ac/v1/ OPAL_AUTH_JWT_ISSUER=https://opal.ac/ OPAL_REPO_WATCHER_ENABLED=0 opal-server run &
```

Comment:
I would make this configurable

Reply:
any reason it is in the same line ? ( makes it harder to read )

Reply:
Why does configurable is meaningful here? Those are app tests. (Basically that part is copied from `run-example-with-security.sh`

Reply:
I'll break it into lines

Reply:
because one might want to test its' own opal server / client and make sure it works